backend/src/utils/prometheusUtils.ts

```diff
@@ -31,7 +31,7 @@ const callPrometheus = async <T>(
   const url = `${host}/api/v1/${queryType}?${query}`;
 
   fastify.log.info(`Prometheus query: ${query}`);
-  return proxyCall(fastify, request, { method: 'GET', url })
+  return proxyCall(fastify, request, { method: 'GET', url, rejectUnauthorized: false })
```

Comment:
Part of the Pipeline proxy stuff was to drop this from the original implementation -- I think it is causing issues with QE clusters -- adding it back.